packages/legend-graph/src/models/metamodels/pure/packageableElements/domain/Package.ts

```diff
@@ -58,7 +57,9 @@ export class Package extends PackageableElement implements Hashable {
   }
 
   addChild(value: PackageableElement): void {
-    addUniqueEntry(this.children, value);
+    // We prefer to push the element to the children array rather than use `addUniqueEntry` for performance reasons.
```

Comment:
```suggestion
    // NOTE: here we directly push the element to the children array without any checks rather than use `addUniqueEntry` to improve performance.
```